.github/dependabot.yaml

```diff
@@ -8,6 +8,8 @@ updates:
       day: sunday
     target-branch: dev
     labels: [catbuffer-parser]
+    commit-message:
+      prefix: '[monorepo] fix'
```

Comment:
1. why monorepo
2. will this pass commit message linter ?

Reply:
1. I noted above that the commit-message prefix field is limited to 15 characters.  ``[catbuffer/parser] fix`` is too long.  Do we want to create different tag?
2. It should create a commit title of ``[monorepo] fix: xxxxxxx``` where xxxxxxx is the original commit title.

Reply:
i would drop fix ?

Reply:
Let me see which ones will fit.